app/controllers/api/v1/keywords_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    class KeywordsController < ApplicationController
+      include ::Pagy::Backend
+      include API::V1::Pagy::JSONAPIConcern
+      include ErrorHandlerConcern
+
+      def index
+        pagy, keywords_list = pagy(keywords)
+
+        if keywords.any?
+          render json: KeywordSerializer.new(keywords_list, pagy_options(pagy)).serializable_hash, status: :ok
```

Comment:
@malparty Can you also try without calling `. serializable_hash`, maybe it works without it as well.